src/main/java/com/aws/greengrass/clientdevices/auth/PermissionEvaluationUtils.java

```diff
@@ -97,6 +98,25 @@ public static boolean isAuthorized(String operation, String resource,
         return false;
     }
 
+    /**
+     * utility method to permission with variable value.
+     *
+     * @param session    current device session
+     * @param groupToPermissionsMap set of permissions for each device group
+     * @return permission map with updated resources
+     */
+    public static Map<String, Set<Permission>> getDevicePolicyPermissions(
```

Comment:
I think we can adjust the naming to make the intent clearer---this isn't doing a get, but rather a map or transform, on the input.    

I neglected to mention on the last PR, but same with `updateResource`, it's doing variable replacement, so the name should reflect that